### PR TITLE
simplify example gradle builds

### PR DESCRIPTION
**Pull request by gavinking**

### CODE REVIEW

**Review by DavideD — COMMENTED** [review]
I get it that you want to simplify the example, but not being able to test them with the latest snapshots worries me.

**Comment by DavideD** [comment]
I suspect we need to have  a separate repositories for the examples that we can update during releases.
This way we can provide users with the simplest project that's not contaminated with what we need for the build of Hibernate Reactive

**Comment by gavinking** [comment]
@DavideD why exactly is it failing? Old version of gradle?

**Comment by DavideD** [comment]
If you rebase and upgrade to the latest ORM Final, it should work.
There were some changes in the latest snapshot that were breaking the build

File: examples/native-sql-example/build.gradle [hunk 1]
```diff
@@ -20,43 +24,8 @@ dependencies {
...
-        if ( project.hasProperty( 'enableJBossSnapshotsRep' ) ) {
-            // Used only for testing with the latest Hibernate ORM snapshots.
-            maven { url 'https://repository.jboss.org/nexus/content/repositories/snapshots' }
-        }
-    }
-    dependencies {
-        classpath "org.hibernate.orm:hibernate-gradle-plugin:${hibernateOrmGradlePluginVersion}"
-    }
-}
-
```
[the remark this answers is not in the document]
Comment (gavinking) [reply examples/native-sql-example/build.gradle:49]: No, I just missed that. Fixed.

[the remark this answers is not in the document]
Comment (DavideD) [reply examples/native-sql-example/build.gradle:49]: But we also need the properties for the repositories, otherwise we won't be able to test locally

**Comment by DavideD** [comment]
Thanks, I've applied the same changes [here](https://github.com/hibernate/hibernate-reactive/pull/1688)

**Pull request closed without merging**